cuqi/sampler/_laplace_approximation.py

```diff
@@ -126,7 +126,10 @@ def Lk_fun(x_k):
 
         # Pre-computations
         self._model = self.target.likelihood.model   
-        self._data = self.target.likelihood.data
+        if hasattr(self.target.model, 'shift'): # if model has a shift take it into account
```

Comment:
Somehow testing for existing of the field doesn't seem so neat to me. We are really testing whether we have a shifted (possibly linear) model, so would it be better to test for the type of self instead?

Another option might be to always have the shift as an attribute, with the value of zero if "unshifted", then the if statement would not be needed, at the price of an unnecessary subtraction.

Reply:
This is now fixed.